eth_tester/utils/backend_testing.py

```diff
@@ -1203,17 +1203,24 @@ def test_log_filter_includes_old_logs(self, eth_tester):
             [EMITTER_ENUM['LogSingleWithIndex'], 1],
         )
 
-        filter_any_id = eth_tester.create_log_filter(from_block=0)
         _call_emitter(
             eth_tester,
             emitter_address,
             'logSingle',
             [EMITTER_ENUM['LogSingleWithIndex'], 2],
         )
 
+        filter_any_id = eth_tester.create_log_filter(
+            from_block=0,
+            to_block=eth_tester.get_block_by_number('latest')['number']
+        )
+
         logs_changes = eth_tester.get_only_filter_changes(filter_any_id)
         logs_all = eth_tester.get_all_filter_logs(filter_any_id)
-        direct_logs_all = eth_tester.get_logs(from_block=0)
+        direct_logs_all = eth_tester.get_logs(
+            from_block=0,
+            to_block=eth_tester.get_block_by_number('latest')['number']
+        )
```

Comment:
I think we should leave in an old test without setting `to_block`, and add a new one that sets `to_block`.

Reply:
sure wll do